PublisherView.js

```diff
@@ -81,6 +81,12 @@ class PublisherView extends React.Component {
   sayHello() {
     OpenTokPublisherViewManager.sayHello();
   }
```

Comment:
sorry.. this is the native lib.. I should remove it from our code! I think it may be around :)